lib/vanagon/platform/deb.rb

```diff
@@ -29,10 +29,15 @@ def generate_package(project) # rubocop:disable Metrics/AbcSize
       # @param workdir [String] working directory to stage the evaluated templates in
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
-      def generate_packaging_artifacts(workdir, name, binding)
+      # @param project [String] name of the project
```

Comment:
`project` isn't the name, it's the `Vanagon::Project` that we're building for

Reply:
Oh whoops!